ic-os/components/init/setup-encryption/guestos/setup-encryption.sh

```diff
@@ -2,17 +2,31 @@
 
 set -e
 
-if [ -e /dev/vda10 ]; then
+if [ -f /boot/config/REDEPLOY ]; then
+    echo "WARNING! Redeploying node."
+
+    # Backup keys
+    cryptsetup luksOpen /dev/vda10 vda10-crypt --key-file /boot/config/store.keyfile
+    sleep 5
+    mkdir /tmp/preserve
+    dd if=/dev/mapper/store-shared--crypto of=/tmp/preserve/crypto.part bs=100M
+    vgchange -an store
+    cryptsetup luksClose vda10-crypt
+
+    # Trigger "redeployment"
+    rm -rf /boot/config/CONFIGURED
```

Comment:
We should use `shred -f -u -z FILENAME` on these files (doesn't work on entire folders unless `find -exec shred ...`)